src/global/tracking/tracking.cc

```diff
@@ -171,5 +173,35 @@ void InitPlugin(JApplication *app) {
             app
             ));
 
+     // linking of reconstructed particles to PID objects
+     ParticlesWithTruthPIDConfig link_cfg {
+       .momentumRelativeTolerance = 100.0, /// Matching momentum effectively disabled
+       .phiTolerance              = 0.1, /// Matching phi tolerance [rad]
+       .etaTolerance              = 0.2, /// Matching eta tolerance
+     };
+
+     app->Add(new JOmniFactoryGeneratorT<ParticlesWithTruthPID_factory>(
+             "ChargedParticlesWithAssociations",
+             {"MCParticles",                                    // edm4hep::MCParticle
+             "CentralCKFTracks",                                // edm4eic::Track
+             },
+             {"ReconstructedChargedWithoutPIDParticles",                  //
```

Comment:
Is it clearer to refer to this as TruthPID then?
```suggestion
             {"ReconstructedChargedWithTruthPIDParticles",                  //
```

Reply:
The truth PID is to be removed when LUTs come. Users can lookup truth MCParticle via associations.